Userland/Libraries/LibWeb/Layout/InlineFormattingContext.cpp

```diff
@@ -234,24 +234,74 @@ void InlineFormattingContext::apply_justification_to_fragments(CSS::TextJustify
     }
 }
 
+class InlineLevelLineWrappingIterator {
```

Comment:
If this is the behavior we always want, can we make it part of `InlineLevelIterator` instead of needing a separate wrapper class?

Reply:
You're right, went too far with a wrapping iterator here. Fixed.